spec/assembly/image/jp2_creator_spec.rb

```diff
@@ -11,6 +11,20 @@
 
   before { cleanup }
 
+  describe '#make_tmp_tiff' do
+    subject(:tiff_file) { creator.send(:make_tmp_tiff) }
+
+    let(:input_path) { 'spec/test_data/color_rgb_srgb_rot90cw.tif' }
+    let(:vips_output) { Vips::Image.new_from_file tiff_file }
+    let(:plum) { [94.0, 58.0, 101.0] }
+
+    context 'when given a tiff with a rotation hint' do
+      it 'rotates it' do
+        expect(vips_output.getpoint(3, 3)).to eq plum
```

Comment:
a test like this should show that the ending state is not the same as the starting state (non-rotated);  would you pleaes add an expect statement for the same point showing that its color is not the same?

Reply:
I think this test does show this, because I wrote the test before I changed the code.  FIrst, I wrote a test that should pass, then I fixed the code until it did pass.